compile.ts

```diff
@@ -575,7 +630,24 @@ class Compiler {
   ): Variable {
     let dest = outs[0] || this.#temp();
     const args: (string | Variable)[] = [];
-    const info = methods[name];
+    let info = methods[name];
+    if (!info && this.subs.has(name)) {
```

Comment:
You're also supposed to be able to recursively call the "main" function of the behavior. Maybe we should add the exported function to subs also, and keep track of which one is the main?